### PR TITLE
Changed text in Button Ideas section

### PR DESCRIPTION
**Pull request by shngt**
Documentation changed - modified README.md and CONTRIBUTING.md as specified in issue

Issue: #535

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,20 +72,7 @@ Please note we have a code of conduct, please follow it in all your interactions
...
-> PS:- we used the unicode character to represent ideas that have and haven't been checked 'cause the normal markdown syntax doesn't work in tables... This one `- []` && `- [x]`
-
-Here's an example of how the syntax looks in markdown format...
-```markdown
-|status  | Button name    | Description      | Submitted by |
-|--------|----------------|------------------|--------------|
-| &#9745;| ~~**Skew Button**~~| A button that skews on hover. |[who did this](https://exampleuser.com) | 
+If you would like to suggest a new button idea, please create a "New Button Idea" Issue.
 
 ```
```
Comment (shahednasser) [inline CONTRIBUTING.md:77]: Please remove this as well

Comment (shngt) [reply CONTRIBUTING.md:77]: This is the line you requested me to add.. removing it would leave the section empty? Just confirming

**Comment by shngt** [comment]
Removed the section altogether, was unsure whether keeping an empty section made sense. Let me know if you want me to add the "Button Ideas" heading back.

File: CONTRIBUTING.md (hunk 1 above)
Comment (shahednasser) [reply CONTRIBUTING.md:73]: I meant there was an extra ``` left from the previous part that you removed that should be removed as well. Please add the sentence that you added before again

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @shngt for doc

**Comment by shngt** [comment]
Thank you! :)

**Pull request merged**